src/components/common/Button/index.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+// also exported from '@storybook/react' if you can deal with breaking changes in 6.1
+import { Story, Meta } from '@storybook/react/types-6-0'
+import { ButtonBase } from '.'
+import { DarkModeThemeToggler } from 'storybook/decorators'
+
+export default {
+  title: 'Common/Button',
+  component: ButtonBase,
+  decorators: [DarkModeThemeToggler],
+  argTypes: {
+    label: { control: 'text' },
+    theme: {
+      control: null,
+    },
+    as: {
+      control: null,
+    },
+    forwardedAs: {
+      control: null,
+    },
```

Comment:
These are styled-component intrinsic props? And you disable controls because we don't need them or because they break on edit (`theme` would breaks as it is an object I imagine)

Reply:
Yeah intrinsic. Not necessary to be able to edit so removed. Theme works as an object but isn't useful enough. In a perfect world I would love a way to edit the theme obj prop in a cleaner way like the `{ control: 'color' }` would work with a flat `backgroundColor` prop.